configure.sh

```diff
@@ -105,6 +105,7 @@ rm -rf build ; ln -sf build-"${TARGET}" build
 CXXFLAGS=${CXXFLAGS} ${CXXFLAGS_COMMON} ${CXXFLAGS_TARGET} ${CXXFLAGS_CIRCT}
 CPPFLAGS=${CPPFLAGS} ${CPPFLAGS_COMMON} ${CPPFLAGS_TARGET} ${CPPFLAGS_LLVM} ${CPPFLAGS_CIRCT}
 CIRCT_PATH=${CIRCT_PATH}
+CIRCT_ENABLED=${CIRCT_ENABLED}
```

Comment:
`HLS_ENABLED`, no? That the HLS backend uses CIRCT is more of a nuance.

Reply:
I changed this in combination with separating `--enable-hls` and `--circt-path`